lib/common/template.js

```diff
@@ -108,5 +108,18 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
         });
     };
 
+    TemplateService.prototype.unlink = function (name, scope) {
+        scope = scope || 'global';
+        return new Promise(function(resolve, reject) {
```

Comment:
FWIW, that didn't work, which is why it is wrapped this way.  The promise would resolve and the database entry would never be removed.